laravel-api/api/User/Requests/UserUpdatePasswordRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Api\User\Requests;
+
+use Infrastructure\Http\ApiRequest;
+
+class UserUpdatePasswordRequest extends ApiRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        return [
+            'user_email' => 'required|email',
+            'token' => 'required',
```

Comment:
`user_email` and `token` - MAYBE add exists if controller doesn't handle bad/expired/wrong email-token pair situation